pkg/graph/resolvers/operational_solution_subtask.go

```diff
@@ -0,0 +1,149 @@
+package resolvers
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/graph/model"
+	"github.com/cmsgov/mint-app/pkg/models"
+
+	"github.com/cmsgov/mint-app/pkg/accesscontrol"
+	"github.com/cmsgov/mint-app/pkg/authentication"
+	"github.com/cmsgov/mint-app/pkg/storage"
+)
+
+// OperationalSolutionSubtasksCreate implements resolver logic to create an operational solution subtask in the database
+func OperationalSolutionSubtasksCreate(
+	logger *zap.Logger,
+	store *storage.Store,
+	inputs []*model.CreateOperationalSolutionSubtaskInput,
+	solutionID uuid.UUID,
+	principal authentication.Principal,
+) ([]*models.OperationalSolutionSubtask, error) {
+	var subtasks []*models.OperationalSolutionSubtask
+
+	for _, input := range inputs {
+		subtask := models.NewOperationalSolutionSubtask(
+			principal.Account().ID,
+			uuid.New(),
+			solutionID,
+			input.Name,
+			input.Status,
+		)
+
+		err := BaseStructPreCreate(logger, subtask, principal, store, true)
+		if err != nil {
+			return nil, err
+		}
+
+		subtasks = append(subtasks, subtask)
+	}
+
+	return store.OperationalSolutionSubtasksCreate(logger, subtasks)
+}
+
+// OperationalSolutionSubtaskGetByID implements the resolver logic to get an operational solution subtask by ID
+func OperationalSolutionSubtaskGetByID(
+	logger *zap.Logger,
+	store *storage.Store,
+	subtaskID uuid.UUID,
+) (*models.OperationalSolutionSubtask, error) {
+	subtask, err := store.OperationalSolutionSubtaskGetByID(logger, subtaskID)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return subtask, err
+}
+
+// OperationalSolutionSubtasksGetBySolutionID implements the resolver logic to
+// get operational solution subtasks by solution ID
+func OperationalSolutionSubtasksGetBySolutionID(
+	logger *zap.Logger,
+	store *storage.Store,
+	principal authentication.Principal,
+	solutionID uuid.UUID,
+) ([]*models.OperationalSolutionSubtask, error) {
+
+	isCollaborator, err := accesscontrol.IsCollaboratorSolutionID(logger, principal, store, solutionID)
```

Comment:
We don't normally block access from viewing a model if the user isn't a collaborator (anyone can view). It's just updating and deleting that normally need that.  I think the access control check should be removed from this function.